client/src/app/components/Header/Header.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import HamburgerMenu from "./MobileMenu/Hamburger";
+
+import "./header.css";
+
+class Input extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      inputBg: "grey"
+    };
+  }
+
+  render() {
+    return (
+      <div
+        className={`${
+          this.state.inputBg === "grey" ? "searchBarGrey" : "searchBarWhite"
+        } input-container`}
+      >
+        <input
+          onFocus={() => {
+            this.setState({ inputBg: "white" });
+          }}
+          onBlur={() => {
+            this.setState({ inputBg: "grey" });
+          }}
+          placeholder="Search for a project"
+          className={`${
+            this.state.inputBg === "grey" ? "searchBarGrey" : "searchBarWhite"
+          }`}
+        />
+        <button>
+          <i class="fas fa-search" />
+        </button>
+      </div>
+    );
+  }
+}
+
+const Nav = () => {
+  return (
+    <div className="nav-container">
+      <Input />
+      <div>
+        <ul>
+          <li>Projects</li>
+          <li>About Us</li>
+        </ul>
+      </div>
+    </div>
+  );
+};
+
+const Logo = () => (
+  <div className="logo">
+    <h1>
+      ProjectCode
+      <i className="fas fa-code logo" />
+    </h1>
+  </div>
+);
+
+export const Header = () => (
```

Comment:
I am not sure if there is any benefit for having Logo and Nav as separate components. They are very small, and putting them elsewhere only makes the JSX more difficult to read. Having big JSX is ok, as long as it's clean. You can add a few lines of comment if it gets bigger or you want to specify what each part does. But generally, you'll have a className such as clasName='Logo' which will make it obvious.

Having a separate component makes sense to me only if that component is more complex, or has state. Such as that SearchBar. In which case that need to be exported from another file, like you did with the HamburgerMenu